nimpy.nim

```diff
@@ -841,6 +844,25 @@ proc nimJsonToPy(node: JsonNode): PPyObject =
       if ret != 0:
         cannotSerializeErr(k)
 
+# Enum handling
+proc nimpyEnumConvert*[T, U](o:T, u: U) : U =
```

Comment:
This doesn't seem to be needed. `nimpyEnumConvert` is never called with 2 arguments.